<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamClass</class>
 <widget class="QMainWindow" name="ExamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Feed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="feedList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Posts</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="postList"/>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dateLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="timeLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Topic subscriptions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="subscriptionList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="subscribeButton">
            <property name="text">
             <string>Subscribe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Exam.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>ExamClass</receiver>
   <slot>addPost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ExamClass</receiver>
   <slot>updatePost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExamClass</receiver>
   <slot>lineChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subscribeButton</sender>
   <signal>clicked()</signal>
   <receiver>ExamClass</receiver>
   <slot>addSubs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPost()</slot>
  <slot>updatePost()</slot>
  <slot>lineChanged()</slot>
  <slot>addSubs()</slot>
 </slots>
</ui>
